charts/rstudio-pm/NEWS.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.24
+
+- Unset the [Server].Address default value as this can cause issues when serving from a domain name.
```

Comment:
```suggestion
- Unset the `[Server].Address` default value as this can cause issues when serving from a domain name.
```
Or `config.Server.Address` which was the format used in previous NEWS items